x-pack/test/functional/apps/lens/group2/table.ts

```diff
@@ -73,7 +73,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       expect(await lens.getDatatableCellText(0, 0)).to.eql('169.228.188.120');
 
       await retry.try(async () => {
-        await lens.changeTableSortingBy(4, 'none');
+        await lens.changeTableSortingBy(4, 'descending');
```

Comment:
Why this change?

Reply:
It looks like there's a couple duplicate questions around this - let's consolidate to one thread. Please see https://github.com/elastic/kibana/pull/192756/commits/4eabb39e05cac7f5960230341ba978761cc2ed8d and https://github.com/elastic/kibana/pull/192756#discussion_r1768637396